docs/configuration/index.md

```diff
@@ -775,6 +775,11 @@ will likely change in the future.
 We will be happy for any feedback regarding it, [please contact us](https://github.com/packit/packit/discussions/2371)!
 :::
 
+#### csmock_args
+
+(*string*) An option to set arguments of `csmock` (the tool used inside [OpenScanHub](https://openscanhub.fedoraproject.org/)). For instance, you can specify arguments like `--cppcheck-add-flag=--enable=style` to check styling issues as seen in [this example run](https://openscanhub.fedoraproject.org/task/9993/) triggered on Avahi with [this pull request change](https://github.com/avahi/avahi/pull/635) being reverted.
```

Comment:
Why not directly mention the pull request (https://github.com/avahi/avahi/pull/641) before referencing the OpenScanHub task?

Reply:
I'll update it. (I've written this before you've created that one.;-)

Reply:
Added. I hope it's clear now.